src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.command.Command;
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+/**
+ * Duke, the personal chatbot that helps a person keep track of
+ * todos, events, deadlines.
+ *
+ * @author Derrick Khoo
+ */
+public class Duke {
+
+    private Storage storage;
+    private Ui ui;
+    private TaskList taskList;
+
+    /**
+     * Constructs an instance of Duke.
+     *
+     * @param filePath the file location for saving data
```

Comment:
Maybe you could consider adding punctuations behind parameter descriptions to better comply with the coding standards? I think this could apply to other parts of your code too.

Referencing the comments section in - https://se-education.org/guides/conventions/java/intermediate.html